pkg/cmd/step/e2e/step_e2e_gc.go

```diff
@@ -114,6 +170,17 @@ func (o *StepE2EGCOptions) Run() error {
 	return gkeGCOpts.Run()
 }
 
+// GetBuildNumberFromClusterEKS gets the build number from the cluster labels
```

Comment:
That's the PR number of the build which triggered the creation of the cluster? Just wondering?